src/Microsoft.DotNet.Wpf/src/PresentationFramework/Microsoft/Win32/SaveFileDialog.cs

```diff
@@ -224,10 +200,9 @@ internal override bool PromptUserIfAppropriate(string fileName)
                 return false;
             }
          
-            bool fExist = File.Exists(Path.GetFullPath(fileName));
+            bool fExist = File.Exists(fileName);
```

Comment:
What if the user writes a relative path in the dialog text box ? Will this still work ?

Reply:
To answer your question yes, see docs for `File.Exists`:

> The path parameter is permitted to specify relative or absolute path information. Relative path information is interpreted as relative to the current working directory.

When you look at the code for `File.Exists`, you will see the first thing it does is it gets the full path: https://referencesource.microsoft.com/#mscorlib/system/io/file.cs,6fef4ff3e625580b